main.cpp

```diff
@@ -79,51 +79,55 @@ void variableDeclarations()
     double secondDouble = 11.5;
     double thirdDouble = 12.000;
     
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
-    ignoreUnused(secondNumber);
-    ignoreUnused(thirdNumber);
-    ignoreUnused(firstInitial);
-    ignoreUnused(lastInitial);
-    ignoreUnused(middleInitial);
-    ignoreUnused(thisIsTrue);
-    ignoreUnused(thisIsFalse);
-    ignoreUnused(isCorrect);
-    ignoreUnused(firstFloat);
-    ignoreUnused(secondFloat);
-    ignoreUnused(thirdFloat);
-    ignoreUnused(firstDouble);
-    ignoreUnused(secondDouble);
-    ignoreUnused(thirdDouble);
+    ignoreUnused
```

Comment:
No, this notation is not recommended.   you would use this notation with curly braces `{ }` if you were initializing an object or some container that took a `std::initializer_list` as a function parameter.
- [ ] revise.